### ISSUE TITLE
Set up cron job for delayed alerts

### ISSUE THREAD

**Issue by stevenday** [mysociety/remit#197]

**Comment by stevenday** [comment]
If I turn on the alerts right now it will send a lot of email to people about studies being delayed. I think we need to check with @kimmwest when it would be a good time to enable this (presumably after the PIs are on-board) and to check what studies it will alert about manually.

**Comment by kimmwest** [comment]
Hey Steve, would be ideal if the system didn't email anyone automatically for a while.  Would there be a way to set it up so only studies added after launch did this?  It will take us some time to actually figure out where each study currently in the system is up to @stevenday

**Comment by stevenday** [comment]
@kimmwest - it would be easier for us just not to email anyone (otherwise we need to change and re-test all the alerting code to ignore existing studies). Would that be ok?

We can then turn the alerting on when you've got a better handle on things.

**Comment by kimmwest** [comment]
Yes you can switch off the alerts for a while - I don't want to put them off a system we haven't got going properly!  @stevenday
